packages/rx_bloc_generator/lib/src/builders/event_arguments_class.dart

```diff
@@ -17,7 +17,7 @@ class _EventArgumentsClass implements _BuilderContract {
         (b) => b
           ..docs.addAll([
             '/// Helps providing the arguments in the [Subject.add] for',
-            '/// [${method.enclosingElement.name ?? ''}.${method.name}] event'
+            '/// [${method.enclosingElement2.name ?? ''}.${method.name}] event'
```

Comment:
What's the reason of changing **enclosingElement** to **enclosingElement2** ?

Reply:
Honestly, beats me why they chose to name the newer field like that. The suggested replacement for `enclosingElement` (which was deprecated) was `enclosingElement2`.